src/pages/instances/forms/InstanceEditDetailsForm.tsx

```diff
@@ -7,7 +7,7 @@ import { EditInstanceFormValues } from "pages/instances/EditInstanceForm";
 export interface InstanceEditDetailsFormValues {
   name: string;
   description?: string;
-  image: string;
+  image?: string;
```

Comment:
This won't fix the current error, as we reference the original image config and not the form image.

I propose to remove image and instanceType completely as both are immutable, see also https://github.com/canonical/lxd-ui/pull/234

Reply:
Good point, I approved #234. I'll close this one then.